eng/illink.targets

```diff
@@ -52,22 +52,28 @@
     <None Include="@(ILLinkSubstitutionsXmls)" />
   </ItemGroup>
 
-  <!-- Custom binplacing for pre/post-trimming and reports that is useful for analysis
+  <!-- Custom binplacing for post-trimming and reports that is useful for analysis
        Must be enabled by setting BinPlaceILLinkTrimAssembly=true
   -->
   <ItemGroup Condition="'$(BinPlaceILLinkTrimAssembly)' == 'true'">
-    <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/trimmed</RuntimePath>
+    <BinPlaceTargetFrameworks Include="$(BuildSettings)">
+      <RuntimePath>$([MSBuild]::NormalizePath('$(LibrariesTrimmingArtifactsRoot)', 'trimmed'))</RuntimePath>
       <ItemName>TrimmedItem</ItemName>
-    </BinPlaceTargetFramework>
-    <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/reports</RuntimePath>
+    </BinPlaceTargetFrameworks>
+    <BinPlaceTargetFrameworks Include="$(BuildSettings)">
+      <RuntimePath>$([MSBuild]::NormalizePath('$(LibrariesTrimmingArtifactsRoot)', 'reports'))</RuntimePath>
       <ItemName>TrimmingReport</ItemName>
-    </BinPlaceTargetFramework>
-    <BinPlaceTargetFramework Include="$(BuildSettings)">
-      <RuntimePath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/pretrimmed</RuntimePath>
+    </BinPlaceTargetFrameworks>
+  </ItemGroup>
+
+  <!-- Binplacing for pre-trimming that is used when running the linker
+       on the entire runtime pack to observe warnings from verbose output
+  -->
+  <ItemGroup>
+    <BinPlaceTargetFrameworks Include="$(NetCoreAppCurrent)">
```

Comment:
Do we need a separate directory for this?  We can't just use the runtimepack itself?

Reply:
I believe the runtime pack contains individually trimmed assemblies. I think they needed non-trimmed assemblies at all.

Reply:
I understand now that we need to run this on the output of the first linker run i.e the runtime pack containing the trimmed assemblies. Will make the change.

Reply:
Great, glad to reduce the number of binplacing targets.